pkg/subroutines/creator_test.go

```diff
@@ -0,0 +1,274 @@
+package subroutines_test
+
+import (
+	"context"
+	"testing"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openmfp/account-operator/api/v1alpha1"
+	"github.com/openmfp/account-operator/pkg/subroutines"
+	"github.com/openmfp/account-operator/pkg/subroutines/mocks"
+)
+
+func TestCreatorSubroutine_GetName(t *testing.T) {
+	routine := subroutines.NewCreatorSubroutine(nil, "1")
+	assert.Equal(t, "CreatorSubroutine", routine.GetName())
+}
+
+func TestCreatorSubroutine_Finalizers(t *testing.T) {
+	routine := subroutines.NewCreatorSubroutine(nil, "1")
+	assert.Equal(t, []string{}, routine.Finalizers())
+}
+
+func TestCreatorSubroutine_Process(t *testing.T) {
+	namespace := "namespace"
+	creator := "test-creator"
+
+	tests := []struct {
+		name        string
+		account     v1alpha1.Account
+		in          *openfgav1.WriteRequest
+		out         *openfgav1.WriteResponse
+		ctx         context.Context
+		setupMocks  func(context.Context, *openfgav1.WriteRequest, *openfgav1.WriteResponse, *mocks.OpenFGAServiceClient)
+		expectError bool
+	}{
+		{
+			name: "should write owner",
```

Comment:
name appears in the console output(in case of failed test for instance) with spaces replaced with underscore(`_`)
Please, replace spaces with underscore in tests as well to make search easier.